README.md

```diff
@@ -49,23 +49,23 @@ What are the types of the following expressions and what do they evaluate to?
 
 ```js
 17
-
+// 17
```

Comment:
remember to read and do the whole problem. These are missing the type description.